diff -up kbd-2.5.1/tests/libkeymap.at.renamed-keymap-in-test kbd-2.5.1/tests/libkeymap.at
--- kbd-2.5.1/tests/libkeymap.at.renamed-keymap-in-test	2022-06-19 04:56:26.986016165 +0200
+++ kbd-2.5.1/tests/libkeymap.at	2022-06-19 12:13:43.162852780 +0200
@@ -173,6 +173,6 @@ AT_SKIP_IF([ test "$(arch)" != "x86_64"
 cp -f -- \
 	"$abs_srcdir/data/bkeymap-2.0.4/cz.map.bin" \
 	expout
-AT_CHECK([$abs_top_builddir/src/loadkeys -b "$abs_srcdir/data/keymaps/i386/qwerty/cz.map"],
+AT_CHECK([$abs_top_builddir/src/loadkeys -b "$abs_srcdir/data/keymaps/i386/qwerty/cz-qwerty.map"],
 	[0], [expout])
 AT_CLEANUP
