diff -Naur kbd-2.0.0/src/unicode_start kbd-2.0.0.tpg/src/unicode_start
--- kbd-2.0.0/src/unicode_start	2014-08-16 16:58:48.000000000 +0000
+++ kbd-2.0.0.tpg/src/unicode_start	2014-08-16 17:01:10.091744753 +0000
@@ -72,6 +72,12 @@
 # have a Unicode map attached, or explicitly specified, e.g.,
 # by giving `def.uni' as a second argument.
 
+# Source /etc/vconsole.conf to get default font
+. /etc/vconsole.conf
+
+DEFAULT_UNICODE_FONT=$FONT
+# Also drdos8x16 is a good candidate.
+
 case "$#" in
 	2)
 		setfont "$1" -u "$2"
@@ -80,6 +86,7 @@
 		setfont "$1"
 		;;
 	0)
+		setfont $DEFAULT_UNICODE_FONT
 		;;
 	*)
 		echo "usage: unicode_start [font [unicode map]]"
