diff -p -up kbd-1.14.1/src/unicode_start.orig kbd-1.14.1/src/unicode_start
--- kbd-1.14.1/src/unicode_start.orig	2008-11-11 15:27:53.000000000 -0200
+++ kbd-1.14.1/src/unicode_start	2008-11-11 15:28:36.000000000 -0200
@@ -34,29 +34,29 @@ kbd_mode -u
 # not the global keymap. root will have to load the keymap in unicode mode
 # explicitly.
 
-uid="`id -u 2>/dev/null`" ||:
-if [ "$uid" = '0' ]; then
-	# There is no way of reverting the effect of "dumpkeys | loadkeys --unicode",
-	# the memory of the earlier keymap is lost. Therefore, try
-	# to save a copy of the original keymap to be able to reload it in unicode_stop.
-	# (see also http://mail.nl.linux.org/linux-utf8/2003-08/msg00053.html):
-
-	[ -n "$HOME" -a "$HOME" != '/' ] ||
-		HOME='/root'
-
-	if [ -d "$HOME" -a -w "$HOME" ]; then
-		[ -d "$HOME/.kbd" ] ||
-			mkdir -- "$HOME/.kbd"
-
-		[ ! -w "$HOME/.kbd" ] ||
-			dumpkeys > "$HOME/.kbd/.keymap_sv"
-	fi
-
-	# redirect stderr and stdout of loadkeys to /dev/null to avoid the confusing
-	# "plus before udiaeresis ignored" warnings.
-
-	dumpkeys | loadkeys --unicode > /dev/null 2>&1
-fi
+#uid="`id -u 2>/dev/null`" ||:
+#if [ "$uid" = '0' ]; then
+#	# There is no way of reverting the effect of "dumpkeys | loadkeys --unicode",
+#	# the memory of the earlier keymap is lost. Therefore, try
+#	# to save a copy of the original keymap to be able to reload it in unicode_stop.
+#	# (see also http://mail.nl.linux.org/linux-utf8/2003-08/msg00053.html):
+#
+#	[ -n "$HOME" -a "$HOME" != '/' ] ||
+#		HOME='/root'
+#
+#	if [ -d "$HOME" -a -w "$HOME" ]; then
+#		[ -d "$HOME/.kbd" ] ||
+#			mkdir -- "$HOME/.kbd"
+#
+#		[ ! -w "$HOME/.kbd" ] ||
+#			dumpkeys > "$HOME/.kbd/.keymap_sv"
+#	fi
+#
+#	# redirect stderr and stdout of loadkeys to /dev/null to avoid the confusing
+#	# "plus before udiaeresis ignored" warnings.
+#
+#	dumpkeys | loadkeys --unicode > /dev/null 2>&1
+#fi
 
 # 2. The output side: the console screen.
 
