--- kbd-1.12.ppc/data/keymaps/i386/include/compose.inc	2007-05-15 17:43:50.000000000 -0300
+++ kbd-1.12/data/keymaps/i386/include/compose.inc	2007-05-15 17:46:25.000000000 -0300
@@ -3,7 +3,6 @@
 #
 # (c) 1999 Bernhard Rosenkränzer <bero@mandrakesoft.com>
 
-keycode  99 = Compose
-keycode 100 = AltGr
-        alt     keycode 100 = Compose
+keycode 125 = AltGr
+        shift     keycode 125 = Compose
 
